A few tests fails after recent changes with keystrokes

## 📝 Provide detailed reproduction steps (if any)

1. Execute: `yarn run test --reporter=dots -f core,basic-styles,link,restricted-editing,undo`

### ✔️ Expected result

Green tests.

### ❌ Actual result

Red.

```
................................................................................
.................................
Chrome 88.0.4324.192 (Mac OS 10.15.7) EditingKeystrokeHandler set() supports priorities FAILED
	AssertError: expected spy, spy, spy, execute to be called in order but were called as
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assertCallOrder [as callOrder] (node_modules/sinon/pkg/sinon.js:188:17)
	    at Context.eval (webpack:///./packages/ckeditor5-core/tests/editingkeystrokehandler.js?:95:17)
Chrome 88.0.4324.192 (Mac OS 10.15.7) EditingKeystrokeHandler set() with a command prevents default when the keystroke was handled FAILED
	AssertError: expected execute to be called with exact arguments
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledWithExactly] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-core/tests/editingkeystrokehandler.js?:34:18)
.
Chrome 88.0.4324.192 (Mac OS 10.15.7) EditingKeystrokeHandler set() with a command provides a callback which stops the event and remaining callbacks in the keystroke handler FAILED
	AssertError: expected spy, spy, spy to be called in order but were called as
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assertCallOrder [as callOrder] (node_modules/sinon/pkg/sinon.js:188:17)
	    at Context.eval (webpack:///./packages/ckeditor5-core/tests/editingkeystrokehandler.js?:64:18)
Chrome 88.0.4324.192 (Mac OS 10.15.7) EditingKeystrokeHandler set() with a callback never prevents default FAILED
	AssertError: expected spy to be called once but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledOnce] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-core/tests/editingkeystrokehandler.js?:77:18)
Chrome 88.0.4324.192 (Mac OS 10.15.7) EditingKeystrokeHandler press() executes a command FAILED
	AssertError: expected execute to be called once but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledOnce] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-core/tests/editingkeystrokehandler.js?:105:17)
Chrome 88.0.4324.192 (Mac OS 10.15.7) EditingKeystrokeHandler press() executes a callback FAILED
	AssertionError: expected false to be true
	    at Context.eval (webpack:///./packages/ckeditor5-core/tests/editingkeystrokehandler.js?:118:39)
................................................................................
................................................................................
.............................................................
Chrome 88.0.4324.192 (Mac OS 10.15.7) BoldEditing should set editor keystroke FAILED
	AssertionError: expected false to be true
	    at Context.eval (webpack:///./packages/ckeditor5-basic-styles/tests/bold/boldediting.js?:78:29)
...............................................
Chrome 88.0.4324.192 (Mac OS 10.15.7) ItalicUI should set editor keystroke FAILED
	AssertionError: expected false to be true
	    at Context.eval (webpack:///./packages/ckeditor5-basic-styles/tests/italic/italicui.js?:96:29)
..................
Chrome 88.0.4324.192 (Mac OS 10.15.7) StrikethroughUI should set editor keystroke FAILED
	AssertionError: expected false to be true
	    at Context.eval (webpack:///./packages/ckeditor5-basic-styles/tests/strikethrough/strikethroughui.js?:98:29)
..............................................
Chrome 88.0.4324.192 (Mac OS 10.15.7) Underline should set editor keystroke FAILED
	AssertionError: expected false to be true
	    at Context.eval (webpack:///./packages/ckeditor5-basic-styles/tests/underline/underlineui.js?:96:29)
................................................................................
................................................................................
................................................................................
................................................................................
.......................................
Chrome 88.0.4324.192 (Mac OS 10.15.7) LinkUI keyboard support should show the UI on Ctrl+K keystroke FAILED
	AssertError: expected _showUI to be called with exact arguments
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledWithExactly] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-link/tests/linkui.js?:813:17)
.
Chrome 88.0.4324.192 (Mac OS 10.15.7) LinkUI keyboard support should prevent default action on Ctrl+K keystroke FAILED
	AssertError: expected spy to be called once but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledOnce] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-link/tests/linkui.js?:842:17)
Chrome 88.0.4324.192 (Mac OS 10.15.7) LinkUI keyboard support should make stack with link visible on Ctrl+K keystroke - no link FAILED
	AssertionError: expected { Object (element, isRendered, ...) } to equal { Object (element, isRendered, ...) }
	    at Context.eval (webpack:///./packages/ckeditor5-link/tests/linkui.js?:863:37)
Chrome 88.0.4324.192 (Mac OS 10.15.7) LinkUI keyboard support should make stack with link visible on Ctrl+K keystroke - link FAILED
	AssertionError: expected { Object (element, isRendered, ...) } to equal { Object (element, isRendered, ...) }
	    at Context.eval (webpack:///./packages/ckeditor5-link/tests/linkui.js?:885:37)
.....................
Chrome 88.0.4324.192 (Mac OS 10.15.7) LinkUI actions view binding should add the #formView upon Ctrl+K keystroke press FAILED
	AssertionError: expected { Object (element, isRendered, ...) } to equal { Object (element, isRendered, ...) }
	    at Context.eval (webpack:///./packages/ckeditor5-link/tests/linkui.js?:1193:38)
................................................................................
................................................................................
................................................................................
.................................................
Chrome 88.0.4324.192 (Mac OS 10.15.7) RestrictedEditingModeEditing custom keydown behaviour Ctrl+A handler collapsed selection should select text only within an exception when selection is inside an exception FAILED
	AssertError: expected spy to be called once but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledOnce] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-restricted-editing/tests/restrictededitingmodeediting.js?:1471:19)
Chrome 88.0.4324.192 (Mac OS 10.15.7) RestrictedEditingModeEditing custom keydown behaviour Ctrl+A handler collapsed selection should select text only within an exception when selection is at the begining of an exception FAILED
	AssertError: expected spy to be called once but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledOnce] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-restricted-editing/tests/restrictededitingmodeediting.js?:1486:19)
Chrome 88.0.4324.192 (Mac OS 10.15.7) RestrictedEditingModeEditing custom keydown behaviour Ctrl+A handler collapsed selection should select text only within an exception when selection is at the end of an exception FAILED
	AssertError: expected spy to be called once but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledOnce] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-restricted-editing/tests/restrictededitingmodeediting.js?:1501:19)
.
Chrome 88.0.4324.192 (Mac OS 10.15.7) RestrictedEditingModeEditing custom keydown behaviour Ctrl+A handler collapsed selection should not extend the selection outside an exception when press Ctrl+A second time FAILED
	AssertError: expected spy to be called twice but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledTwice] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-restricted-editing/tests/restrictededitingmodeediting.js?:1530:19)
Chrome 88.0.4324.192 (Mac OS 10.15.7) RestrictedEditingModeEditing custom keydown behaviour Ctrl+A handler non-collapsed selection should select text within an exception when a whole selection range is inside an exception FAILED
	AssertError: expected spy to be called once but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledOnce] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-restricted-editing/tests/restrictededitingmodeediting.js?:1547:19)
Chrome 88.0.4324.192 (Mac OS 10.15.7) RestrictedEditingModeEditing custom keydown behaviour Ctrl+A handler non-collapsed selection should select text within an exception when end of selection range is equal exception end FAILED
	AssertError: expected spy to be called once but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledOnce] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-restricted-editing/tests/restrictededitingmodeediting.js?:1562:19)
Chrome 88.0.4324.192 (Mac OS 10.15.7) RestrictedEditingModeEditing custom keydown behaviour Ctrl+A handler non-collapsed selection should select text within an exception when start of selection range is equal exception start FAILED
	AssertError: expected spy to be called once but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledOnce] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-restricted-editing/tests/restrictededitingmodeediting.js?:1577:19)
.
Chrome 88.0.4324.192 (Mac OS 10.15.7) RestrictedEditingModeEditing custom keydown behaviour Ctrl+A handler non-collapsed selection should not extend the selection outside an exception when press Ctrl+A second time FAILED
	AssertError: expected spy to be called twice but was called 0 times
	    at Object.fail (node_modules/sinon/pkg/sinon.js:158:25)
	    at failAssertion (node_modules/sinon/pkg/sinon.js:115:20)
	    at Object.assert.<computed> [as calledTwice] (node_modules/sinon/pkg/sinon.js:141:17)
	    at Context.eval (webpack:///./packages/ckeditor5-restricted-editing/tests/restrictededitingmodeediting.js?:1608:19)
................................................................................
..................................
Chrome 88.0.4324.192 (Mac OS 10.15.7) UndoEditing should set CTRL+Z keystroke FAILED
	AssertionError: expected false to be true
	    at Context.eval (webpack:///./packages/ckeditor5-undo/tests/undoediting.js?:155:29)
Chrome 88.0.4324.192 (Mac OS 10.15.7) UndoEditing should set CTRL+Y keystroke FAILED
	AssertionError: expected false to be true
	    at Context.eval (webpack:///./packages/ckeditor5-undo/tests/undoediting.js?:169:29)
Chrome 88.0.4324.192 (Mac OS 10.15.7) UndoEditing should set CTRL+SHIFT+Z keystroke FAILED
	AssertionError: expected false to be true
	    at Context.eval (webpack:///./packages/ckeditor5-undo/tests/undoediting.js?:185:29)
..............
Chrome 88.0.4324.192 (Mac OS 10.15.7): Executed 1272 of 1272 (26 FAILED) (21.126 secs / 4.527 secs)
```

## 📃 Other details

* Started failing after merging #9057
* Works fine for a commit before the merge: 959c1d6d56d43468f01afed6c27637a449f78515
* OS: **MacOS**
* First affected CKEditor version: 8dac3a98bb93cc6e1d0bfa8d2db8a5d9a6f89988
* Installed CKEditor plugins: `#master`

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

--- a/packages/ckeditor5-basic-styles/tests/bold/boldediting.js
+++ b/packages/ckeditor5-basic-styles/tests/bold/boldediting.js
@@ -12,6 +12,7 @@ import AttributeCommand from '../../src/attributecommand';
 import { getData as getModelData, setData as setModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
 import { getData as getViewData } from '@ckeditor/ckeditor5-engine/src/dev-utils/view';
 import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
+import env from '@ckeditor/ckeditor5-utils/src/env';
 
 describe( 'BoldEditing', () => {
 	let editor, model;
@@ -60,7 +61,8 @@ describe( 'BoldEditing', () => {
 		const spy = sinon.spy( editor, 'execute' );
 		const keyEventData = {
 			keyCode: keyCodes.b,
-			ctrlKey: true,
+			ctrlKey: !env.isMac,
+			metaKey: env.isMac,
 			preventDefault: sinon.spy(),
 			stopPropagation: sinon.spy()
 		};

--- a/packages/ckeditor5-basic-styles/tests/italic/italicui.js
+++ b/packages/ckeditor5-basic-styles/tests/italic/italicui.js
@@ -10,6 +10,7 @@ import ItalicEditing from '../../src/italic/italicediting';
 import ItalicUI from '../../src/italic/italicui';
 import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
 import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils';
+import env from '@ckeditor/ckeditor5-utils/src/env';
 import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
 
 import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
@@ -80,7 +81,8 @@ describe( 'ItalicUI', () => {
 
 		const wasHandled = editor.keystrokes.press( {
 			keyCode: keyCodes.i,
-			ctrlKey: true,
+			ctrlKey: !env.isMac,
+			metaKey: env.isMac,
 			preventDefault: sinon.spy(),
 			stopPropagation: sinon.spy()
 		} );

--- a/packages/ckeditor5-basic-styles/tests/strikethrough/strikethroughui.js
+++ b/packages/ckeditor5-basic-styles/tests/strikethrough/strikethroughui.js
@@ -10,6 +10,7 @@ import StrikethroughEditing from '../../src/strikethrough/strikethroughediting';
 import StrikethroughUI from '../../src/strikethrough/strikethroughui';
 import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
 import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils';
+import env from '@ckeditor/ckeditor5-utils/src/env';
 import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
 
 import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
@@ -80,7 +81,8 @@ describe( 'StrikethroughUI', () => {
 		const keyEventData = {
 			keyCode: keyCodes.x,
 			shiftKey: true,
-			ctrlKey: true,
+			ctrlKey: !env.isMac,
+			metaKey: env.isMac,
 			preventDefault: sinon.spy(),
 			stopPropagation: sinon.spy()
 		};

--- a/packages/ckeditor5-basic-styles/tests/underline/underlineui.js
+++ b/packages/ckeditor5-basic-styles/tests/underline/underlineui.js
@@ -10,6 +10,7 @@ import UnderlineEditing from '../../src/underline/underlineediting';
 import UnderlineUI from '../../src/underline/underlineui';
 import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
 import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils';
+import env from '@ckeditor/ckeditor5-utils/src/env';
 import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
 
 import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
@@ -80,7 +81,8 @@ describe( 'Underline', () => {
 
 		const wasHandled = editor.keystrokes.press( {
 			keyCode: keyCodes.u,
-			ctrlKey: true,
+			ctrlKey: !env.isMac,
+			metaKey: env.isMac,
 			preventDefault: sinon.spy(),
 			stopPropagation: sinon.spy()
 		} );

--- a/packages/ckeditor5-core/tests/editingkeystrokehandler.js
+++ b/packages/ckeditor5-core/tests/editingkeystrokehandler.js
@@ -6,6 +6,7 @@
 import VirtualTestEditor from '../tests/_utils/virtualtesteditor';
 import EditingKeystrokeHandler from '../src/editingkeystrokehandler';
 import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
+import env from '@ckeditor/ckeditor5-utils/src/env';
 
 describe( 'EditingKeystrokeHandler', () => {
 	let editor, keystrokes, executeSpy;
@@ -120,7 +121,8 @@ describe( 'EditingKeystrokeHandler', () => {
 function getCtrlA() {
 	return {
 		keyCode: keyCodes.a,
-		ctrlKey: true,
+		ctrlKey: !env.isMac,
+		metaKey: env.isMac,
 		preventDefault: sinon.spy(),
 		stopPropagation: sinon.spy()
 	};

--- a/packages/ckeditor5-link/tests/linkui.js
+++ b/packages/ckeditor5-link/tests/linkui.js
@@ -12,6 +12,7 @@ import isRange from '@ckeditor/ckeditor5-utils/src/dom/isrange';
 import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
 import { getData as getModelData, setData as setModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
 import { getData as getViewData } from '@ckeditor/ckeditor5-engine/src/dev-utils/view';
+import env from '@ckeditor/ckeditor5-utils/src/env';
 
 import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
 import BlockQuote from '@ckeditor/ckeditor5-block-quote/src/blockquote';
@@ -788,7 +789,8 @@ describe( 'LinkUI', () => {
 
 			editor.keystrokes.press( {
 				keyCode: keyCodes.k,
-				ctrlKey: true,
+				ctrlKey: !env.isMac,
+				metaKey: env.isMac,
 				preventDefault: sinon.spy(),
 				stopPropagation: sinon.spy()
 			} );
@@ -802,7 +804,8 @@ describe( 'LinkUI', () => {
 
 			editor.keystrokes.press( {
 				keyCode: keyCodes.k,
-				ctrlKey: true,
+				ctrlKey: !env.isMac,
+				metaKey: env.isMac,
 				preventDefault: sinon.spy(),
 				stopPropagation: sinon.spy()
 			} );
@@ -816,7 +819,8 @@ describe( 'LinkUI', () => {
 
 			editor.keystrokes.press( {
 				keyCode: keyCodes.k,
-				ctrlKey: true,
+				ctrlKey: !env.isMac,
+				metaKey: env.isMac,
 				preventDefault: preventDefaultSpy,
 				stopPropagation: stopPropagationSpy
 			} );
@@ -837,7 +841,8 @@ describe( 'LinkUI', () => {
 
 			editor.keystrokes.press( {
 				keyCode: keyCodes.k,
-				ctrlKey: true,
+				ctrlKey: !env.isMac,
+				metaKey: env.isMac,
 				preventDefault: sinon.spy(),
 				stopPropagation: sinon.spy()
 			} );
@@ -859,7 +864,8 @@ describe( 'LinkUI', () => {
 
 			editor.keystrokes.press( {
 				keyCode: keyCodes.k,
-				ctrlKey: true,
+				ctrlKey: !env.isMac,
+				metaKey: env.isMac,
 				preventDefault: sinon.spy(),
 				stopPropagation: sinon.spy()
 			} );
@@ -868,7 +874,8 @@ describe( 'LinkUI', () => {
 
 			editor.keystrokes.press( {
 				keyCode: keyCodes.k,
-				ctrlKey: true,
+				ctrlKey: !env.isMac,
+				metaKey: env.isMac,
 				preventDefault: sinon.spy(),
 				stopPropagation: sinon.spy()
 			} );
@@ -1162,7 +1169,8 @@ describe( 'LinkUI', () => {
 			it( 'should add the #formView upon Ctrl+K keystroke press', () => {
 				const keyEvtData = {
 					keyCode: keyCodes.k,
-					ctrlKey: true,
+					ctrlKey: !env.isMac,
+					metaKey: env.isMac,
 					preventDefault: sinon.spy(),
 					stopPropagation: sinon.spy()
 				};

--- a/packages/ckeditor5-restricted-editing/tests/restrictededitingmodeediting.js
+++ b/packages/ckeditor5-restricted-editing/tests/restrictededitingmodeediting.js
@@ -23,6 +23,7 @@ import ItalicEditing from '@ckeditor/ckeditor5-basic-styles/src/italic/italicedi
 import BlockQuoteEditing from '@ckeditor/ckeditor5-block-quote/src/blockquoteediting';
 import TableEditing from '@ckeditor/ckeditor5-table/src/tableediting';
 import Command from '@ckeditor/ckeditor5-core/src/command';
+import env from '@ckeditor/ckeditor5-utils/src/env';
 
 describe( 'RestrictedEditingModeEditing', () => {
 	let editor, model;
@@ -1432,7 +1433,8 @@ describe( 'RestrictedEditingModeEditing', () => {
 			beforeEach( async () => {
 				evtData = {
 					keyCode: getCode( 'A' ),
-					ctrlKey: true,
+					ctrlKey: !env.isMac,
+					metaKey: env.isMac,
 					preventDefault: sinon.spy(),
 					stopPropagation: sinon.spy()
 				};

--- a/packages/ckeditor5-select-all/tests/selectallediting.js
+++ b/packages/ckeditor5-select-all/tests/selectallediting.js
@@ -6,6 +6,7 @@
 import ModelTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/modeltesteditor';
 import SelectAllEditing from '../src/selectallediting';
 import SelectAllCommand from '../src/selectallcommand';
+import env from '@ckeditor/ckeditor5-utils/src/env';
 import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
 
 describe( 'SelectAllEditing', () => {
@@ -42,7 +43,8 @@ describe( 'SelectAllEditing', () => {
 		it( 'should execute the "selectAll" command', () => {
 			const domEventDataMock = {
 				keyCode: keyCodes.a,
-				ctrlKey: true,
+				ctrlKey: !env.isMac,
+				metaKey: env.isMac,
 				preventDefault: sinon.spy()
 			};
 
@@ -55,7 +57,8 @@ describe( 'SelectAllEditing', () => {
 		it( 'should prevent the default action', () => {
 			const domEventDataMock = {
 				keyCode: keyCodes.a,
-				ctrlKey: true,
+				ctrlKey: !env.isMac,
+				metaKey: env.isMac,
 				preventDefault: sinon.spy()
 			};
 
@@ -67,7 +70,8 @@ describe( 'SelectAllEditing', () => {
 		it( 'should not react to other keystrokes', () => {
 			const domEventDataMock = {
 				keyCode: keyCodes.x,
-				ctrlKey: true,
+				ctrlKey: !env.isMac,
+				metaKey: env.isMac,
 				preventDefault: sinon.spy()
 			};
 

--- a/packages/ckeditor5-undo/tests/undoediting.js
+++ b/packages/ckeditor5-undo/tests/undoediting.js
@@ -6,6 +6,7 @@
 import ModelTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/modeltesteditor';
 
 import UndoEditing from '../src/undoediting';
+import env from '@ckeditor/ckeditor5-utils/src/env';
 import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
 
 describe( 'UndoEditing', () => {
@@ -143,7 +144,8 @@ describe( 'UndoEditing', () => {
 
 		const wasHandled = editor.keystrokes.press( {
 			keyCode: keyCodes.z,
-			ctrlKey: true,
+			ctrlKey: !env.isMac,
+			metaKey: env.isMac,
 			preventDefault: sinon.spy(),
 			stopPropagation: sinon.spy()
 		} );
@@ -157,7 +159,8 @@ describe( 'UndoEditing', () => {
 
 		const wasHandled = editor.keystrokes.press( {
 			keyCode: keyCodes.y,
-			ctrlKey: true,
+			ctrlKey: !env.isMac,
+			metaKey: env.isMac,
 			preventDefault: sinon.spy(),
 			stopPropagation: sinon.spy()
 		} );
@@ -170,7 +173,8 @@ describe( 'UndoEditing', () => {
 		const spy = sinon.stub( editor, 'execute' );
 		const keyEventData = {
 			keyCode: keyCodes.z,
-			ctrlKey: true,
+			ctrlKey: !env.isMac,
+			metaKey: env.isMac,
 			shiftKey: true,
 			preventDefault: sinon.spy(),
 			stopPropagation: sinon.spy()

